File: CSIT111/Lecture_2/Lecture_Labs/Lecturer_Examples/Circle_4.py
import math
import random

class Circle_4:
    def main(self):
        
		# Generate a random value and assign to radius
		# This value is between 0 and 100
        radius = random.random() * 100
        
        # Compute its area and its perimeter
        area = math.pi * radius * radius
        perimeter = 2.0 * math.pi * radius
        
        # Display the results
        print("Given radius = ", radius)
        print("Area = ", area)
        print("Perimeter = ", perimeter)
        
circle_4 = Circle_4()
circle_4.main()

File: CSIT111/Lecture_2/Lecture_Labs/Lecturer_Examples/Circle_4.java
// Given a radius of a circle.
// - We wish to compute its area and its perimeter.
// - Display the info.
//
// Explore the use of some methods defined in the Math class
// For example, PI belongs to the Math class
//
// File name: Circle_4a.java
//
// Explore the use of static import in our program
import static java.lang.Math.*;
import static java.lang.System.*;

class Circle_4a
{
	public static void main (String [ ] args)
	{
		// Declare variables
		double radius;
		double area, perimeter;
		
		// Generate a random value and assign to radius
		// This value is between 0 and 100
		radius = random () * 100;
		
		// Compute its area and its perimeter
		area = PI * radius * radius;
		perimeter = 2.0 * PI * radius;
		
		// Display the results
		out.print ("Given radius = ");
		out.println (radius);
		
		out.print ("Area = ");
		out.println (area);
		
		out.print ("Perimeter = ");
		out.println (perimeter);
	}
}